--- origsrc/tuxpaint-0.9.21/Makefile	2009-04-29 00:18:57.000000000 -0500
+++ src/tuxpaint-0.9.21/Makefile	2009-11-23 23:03:13.561287900 -0600
@@ -698,7 +698,7 @@ install-bin:
 	@echo
 	@echo "...Installing program itself..."
 	@install -d $(BIN_PREFIX)
-	@cp tuxpaint $(BIN_PREFIX)
+	@cp tuxpaint$(EXE_EXT) $(BIN_PREFIX)
 	@chmod a+rx,g-w,o-w $(BIN_PREFIX)/tuxpaint$(EXE_EXT)
 
 # Install the required Windows DLLs into the 'bdist' directory
