--- origsrc/tuxpaint-0.9.19/src/im.c	2008-02-14 09:44:17.000000000 -0600
+++ src/tuxpaint-0.9.19/src/im.c	2008-05-18 18:02:31.187500000 -0500
@@ -576,7 +576,7 @@
         if(charmap_add(cm, section, buf, unicode, flag)) {
           size_t i = 0;
 
-#ifndef __BEOS__
+#if !defined(__BEOS__) && !defined(__CYGWIN__)
           fwprintf(stderr, L"Unable to add sequence '%ls', unicode ", buf);
           for(i = 0; i < wcslen(unicode); i++) fwprintf(stderr, L"%04X ", (int)unicode[i]);
           fwprintf(stderr, L"in section %d\n", section);
